zerver/webhooks/bitbucket2/view.py

```diff
@@ -198,7 +198,7 @@ def get_force_push_body(payload: Dict[str, Any], change: Dict[str, Any]) -> str:
 
 def get_commit_author_name(commit: Dict[str, Any]) -> str:
     if commit['author'].get('user'):
-        return commit['author']['user'].get('username')
+        return commit['author']['user'].get('display_name')
```

Comment:
Hmm, I'd suggest re-writing these three lines to be:
```python3
author_info = commit['author'].get('user')
if author_info:
    if 'username' in author_info:
        return author_info['username']
    return author_info['display_name']
```
This way we use `display_name` only if the `username` does not exist.